packages/sign-client/src/controllers/engine.ts

```diff
@@ -642,7 +653,12 @@ export class Engine extends IEngine {
       resources.push(recap);
     }
 
-    const expiryTimestamp = calcExpiry(ENGINE_RPC_OPTS.wc_sessionPropose.req.ttl);
+    // ensure the expiry is at least the minimum required for the request - currently 1h
```

Comment:
```suggestion
    // Ensure the expiry is greater than the minimum required for the request - currently 1h
```

Alternatively we can leave the comment as-is if condition uses `>=` but seems pointless if it can just default to the internal `TTL` anyways if equal.